imsv-docs-docusaurus/openapi/endpoints/currency/currency-convert.yaml

```diff
@@ -1,7 +1,8 @@
 get:
   tags:
     - currency
-  summary: Currency conversion
+  summary: Currency Conversion
```

Comment:
`Convert Currency`? not sure if this one is an exception or not

Reply:
agreed, renamed to "Get Currency Conversion"